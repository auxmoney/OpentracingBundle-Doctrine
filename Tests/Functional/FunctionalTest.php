<?php

declare(strict_types=1);

namespace Auxmoney\OpentracingDoctrineDBALBundle\Tests\Functional;

use Auxmoney\OpentracingBundle\Tests\Functional\JaegerFunctionalTest;

class FunctionalTest extends JaegerFunctionalTest
{
    public function testStandard(): void
    {
        $this->setUpTestProject('dbal-standard');

        $this->runInTestProject(['symfony', 'console', 'test:doctrine:setup-table']);
        $this->runInTestProject(['symfony', 'console', 'test:doctrine:tracing']);
        self::assertSame(true, true);
    }
}
