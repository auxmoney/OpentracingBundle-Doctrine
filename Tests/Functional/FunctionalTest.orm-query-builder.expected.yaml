operationName: 'test:doctrine:orm-query-builder'
tags:
  -
    key: command.exit-code
    value: 0
children:
  -
    operationName: 'DBAL: SELECT test_table'
    tags:
      -
        key: db.statement
        value: 'SELECT COUNT(t0_.id) AS sclr_0 FROM test_table t0_ WHERE t0_.str IS NOT NULL'
      -
        key: db.parameters
        value: '[]'
  -
    operationName: 'DBAL: SELECT test_table'
    tags:
      -
        key: db.statement
        value: 'SELECT COUNT(*) FROM test_table t0'
      -
        key: db.parameters
        value: '[]'
  -
    operationName: 'DBAL: TRANSACTION'
    tags:
      -
        key: db.transaction.end
        value: commit
    children:
      -
        operationName: 'DBAL: INSERT INTO test_table'
        tags:
          -
            key: db.statement
            value: 'INSERT INTO test_table (str) VALUES (?)'
          -
            key: db.parameters
            value: '{"1":"a"}'
  -
    operationName: 'DBAL: SELECT test_table'
    tags:
      -
        key: db.statement
        value: 'SELECT COUNT(t0_.id) AS sclr_0 FROM test_table t0_ WHERE t0_.str IS NOT NULL'
      -
        key: db.parameters
        value: '[]'
  -
    operationName: 'DBAL: SELECT test_table'
    tags:
      -
        key: db.statement
        value: 'SELECT COUNT(*) FROM test_table t0'
      -
        key: db.parameters
        value: '[]'
  -
    operationName: 'DBAL: TRANSACTION'
    tags:
      -
        key: db.transaction.end
        value: commit
    children:
      -
        operationName: 'DBAL: INSERT INTO test_table'
        tags:
          -
            key: db.statement
            value: 'INSERT INTO test_table (str) VALUES (?)'
          -
            key: db.parameters
            value: '{"1":"b"}'
  -
    operationName: 'DBAL: SELECT test_table'
    tags:
      -
        key: db.statement
        value: 'SELECT COUNT(t0_.id) AS sclr_0 FROM test_table t0_ WHERE t0_.str IS NOT NULL'
      -
        key: db.parameters
        value: '[]'
  -
    operationName: 'DBAL: SELECT test_table'
    tags:
      -
        key: db.statement
        value: 'SELECT COUNT(*) FROM test_table t0'
      -
        key: db.parameters
        value: '[]'
  -
    operationName: 'DBAL: UPDATE test_table'
    tags:
      -
        key: db.statement
        value: 'UPDATE test_table SET str = NULL WHERE str = ?'
      -
        key: db.parameters
        value: '{"1":"a"}'
  -
    operationName: 'DBAL: SELECT test_table'
    tags:
      -
        key: db.statement
        value: 'SELECT COUNT(t0_.id) AS sclr_0 FROM test_table t0_ WHERE t0_.str IS NOT NULL'
      -
        key: db.parameters
        value: '[]'
  -
    operationName: 'DBAL: SELECT test_table'
    tags:
      -
        key: db.statement
        value: 'SELECT COUNT(*) FROM test_table t0'
      -
        key: db.parameters
        value: '[]'
  -
    operationName: 'DBAL: DELETE test_table'
    tags:
      -
        key: db.statement
        value: 'DELETE FROM test_table WHERE id = 2'
      -
        key: db.parameters
        value: '[]'
  -
    operationName: 'DBAL: SELECT test_table'
    tags:
      -
        key: db.statement
        value: 'SELECT COUNT(t0_.id) AS sclr_0 FROM test_table t0_ WHERE t0_.str IS NOT NULL'
      -
        key: db.parameters
        value: '[]'
  -
    operationName: 'DBAL: SELECT test_table'
    tags:
      -
        key: db.statement
        value: 'SELECT COUNT(*) FROM test_table t0'
      -
        key: db.parameters
        value: '[]'
